### ISSUE TITLE
Missing Thai font

### ISSUE THREAD

**Issue by vazw** [damus-io/notedeck#283]
Hi, I'm just tested notedeck and found that it's missing thai fonts

![Missing Font](https://github.com/user-attachments/assets/ae0083fd-a717-479e-840b-d5f5f12288e7)

I've try added `NotoSansThai` Font into the project and it's look great.

![Added Font](https://github.com/user-attachments/assets/ec830574-17f7-4b2b-b1f0-65eb8df85def)
here is my patch if you agree to apply them I can make a pull request later

```git
diff --git a/src/fonts.rs b/src/fonts.rs
index 5cf3821..9e8d863 100644
--- a/src/fonts.rs
+++ b/src/fonts.rs
@@ -99,11 +99,24 @@ pub fn setup_fonts(ctx: &egui::Context) {
             },
         ),
     );
+    font_data.insert(
+        "NotoSansThai".to_owned(),
+        FontData::from_static(include_bytes!(
+            "../assets/fonts/NotoSansThai-VariableFont.ttf"
+        ))
+        .tweak(FontTweak {
+            scale: 1.1, // make them a touch larger
+            y_offset_factor: 0.0,
+            y_offset: 0.0,
+            baseline_offset_factor: 0.0,
+        }),
+    );

     let base_fonts = vec![
         "DejaVuSans".to_owned(),
         "NotoEmoji".to_owned(),
         "NotoSansCJK".to_owned(),
+        "NotoSansThai".to_owned(),
     ];

     let mut proportional = vec!["Onest".to_owned()];
```

**Comment by jb55** [comment]
should be easy to fix! thanks for catching this.

**Comment by jb55** [comment]
whats your account? I want to test

**Comment by jb55** [comment]
thanks! fixed in 989f88c989f5d6ec85f719efab6f80fb1a95ee00
